test/OpenTelemetry.Contrib.Instrumentation.Remoting.Tests/OpenTelemetry.Contrib.Instrumentation.Remoting.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Unit test project for OpenTelemetry .NET Remoting instrumentation</Description>
+    <TargetFrameworks>net452;net46</TargetFrameworks>
```

Comment:
since we don't have any specific code for net46, we could remove it.

Reply:
Done. Removed from both the tests and the instrumentation projects. No net46-specific code in either.

Reply:
thank you!